server-side-rendering/src/lib/server/appwrite.ts

```diff
@@ -0,0 +1,44 @@
+import type { Cookies, RequestEvent } from '@sveltejs/kit';
+import { Client, Account, Users } from 'luke-node-appwrite-ssr';
+import { APPWRITE_KEY } from '$env/static/private';
+import { PUBLIC_APPWRITE_ENDPOINT } from '$env/static/public';
+
+export const SESSION_COOKIE = 'a_session';
```

Comment:
I dont know why we have it but this doesnt feel right. If its appwrite cookie name we should maybe have const for it in SDK. Or find a different way to get it

Reply:
After reading all PR, I still dont know why we need this. I think this is not related to Appwrite cookie name. let's call it something else, for a second it scared me that we are still parsing and proxying some cookie header. We can call it `auth` or `session` or `secret` or some combination.

Reply:
This is not an Appwrite cookie, this is the server cookie. It can be set to anything you like. The fact the the server uses Appwrite to manage authentication is not important to the client. I've picked `a_session` because this is similiar to how we call session cookie in appwrite. But happens just `session` would be clearer.